### ISSUE TITLE
Backtick + javascript newline escape = error: reserved word "var"

### ISSUE THREAD

**Issue by yetanotherusernamebecausegithubisbugged** [jashkenas/coffeescript#3760]
Backticked some old code to start step-by-step rewriting it in coffeescript and have discovered this confusing error message.

``` coffeescript
`
var str = "long\
string";
`
```

```
$ coffee -p test97.coffee 
Running node v0.11.13
/home/me/tst/test97.coffee:2:1: error: reserved word "var"
var str = "long\
^
```

**Comment by SaranNV** [comment]
hi yetanotherusernamebecausegithubisbugged ,
  I also have this same issue on javascript.i dont know how to fix it..

**Comment by yetanotherusernamebecausegithubisbugged** [comment]
Try to remove all the newline slashes. As long as I remember, it'll help.

**Comment by SaranNV** [comment]
hi yetanotherusernamebecausegithubisbugged,

This is the below error line in the javascript (Gruntfile.js).it showing error as var error:reserved word error
var coffeeFiles = [
  'javascripts/hub_client/augury.js.coffee',
  'javascripts/hub_client/common/**/*.coffee',
  'javascripts/hub_client/extensions/**/*.coffee',
  'javascripts/hub_client/mixins/*.coffee',
  'javascripts/hub_client/events/**/*.coffee',
  'javascripts/hub_client/models/**/*.coffee',
  'javascripts/hub_client/collections/**/*.coffee',
  'javascripts/hub_client/routers/**/*.coffee',
  'javascripts/hub_client/templates/**/*.coffee',
  'javascripts/hub_client/views/**/*.coffee'
]

**Comment by yetanotherusernamebecausegithubisbugged** [comment]
It compiles for me. Probably you have forgot to put a backticks around the javascript code?
```
`
var coffeeFiles = [
'javascripts/hub_client/augury.js.coffee',
'javascripts/hub_client/common//*.coffee',
'javascripts/hub_client/extensions//.coffee',
'javascripts/hub_client/mixins/.coffee',
'javascripts/hub_client/events//*.coffee',
'javascripts/hub_client/models//.coffee',
'javascripts/hub_client/collections//.coffee',
'javascripts/hub_client/routers//*.coffee',
'javascripts/hub_client/templates//.coffee',
'javascripts/hub_client/views//.coffee'
]
`
```
?

**Comment by SaranNV** [comment]
The backticks does not help me.

**Comment by GeoffreyBooth** [comment]
Fixed by #4357
